scylla/src/frame/response/result.rs

```diff
@@ -1,12 +1,13 @@
 use crate::cql_to_rust::{FromRow, FromRowError};
 use crate::frame::response::event::SchemaChangeEvent;
-use crate::frame::value::Counter;
+use crate::frame::types::vint_decode;
+use crate::frame::value::{Counter, CqlDuration};
 use crate::frame::{frame_errors::ParseError, types};
 use bigdecimal::BigDecimal;
 use byteorder::{BigEndian, ReadBytesExt};
 use bytes::{Buf, Bytes};
+use chrono;
 use chrono::prelude::*;
-use chrono::Duration;
 use num_bigint::BigInt;
```

Comment:
Changing the `result.rs` file to use the fully-qualified `chrono::Duration` instead of importing the `Duration` type directly is a refactor which should be placed in its separate commit.